### PR TITLE
Add docker deployment for capstone/files

### PR DESCRIPTION
**Pull request by niedziol**

### CODE REVIEW

**Review by karpp — APPROVED** [review]
LGTM

File: capstone/files/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+echo -e "${YELLOW}Build docker image...${NC}"
+docker build -t files-app .
+echo -e "${GREEN}Image built!${NC}"
+
+echo -e "${YELLOW}Save image...${NC}"
+docker save -o ./files-app-image files-app
+echo -e "${GREEN}Image saved!${NC}"
+
+echo -e "${YELLOW}Upload to server...${NC}"
+scp files-app-image wazuh-agent:~
```
Comment (mgmacias95) [inline capstone/files/deploy.sh:18]: Add a readme or at least a comment specifying the necessary SSH configuration to run this.

Comment (niedziol) [reply capstone/files/deploy.sh:18]: Done

**Review by mgmacias95 — APPROVED** [review]
LGTM

**Pull request merged**